src/Microsoft.ML.Tokenizers/PACKAGE.md

```diff
@@ -0,0 +1,57 @@
+## About
+
+Microsoft.ML.Tokenizers supports various the implmentation of the tokenization used in the NLP transforms.
+
+## Key Features
+
+* Extensisble tokenizer architecture that allows for specialization of Normalizer, PreTokenizer, Model/Encoder, Decoder
+* BPE - Byte pair encoding model
+* English Roberta model
+* Tiktoken model
+
+## How to Use
+
+```c#
+using Microsoft.ML.Tokenizers;
+
+// initialize the tokenizer for `gpt-4` model, downloading data files
+Tokenizer tokenizer = await Tiktoken.CreateByModelNameAsync("gpt-4");
+
+string source = "Text tokenization is the process of splitting a string into a list of tokens.";
+
+Console.WriteLine($"Tokens: {tokenizer.CountTokens(source)}");
+// print: Tokens: 16
+
+var trimIndex = tokenizer.LastIndexOfTokenCount(source, 5, out string processedText, out _);
```

Comment:
I suggest adding the call returning the Ids too. This will be very useful as it is used in the common scenario:

```C#
IReadOnlyList<int> ids = tokenizer.EncodeToIds(processedText);
Console.WriteLine(string.Join(", ", ids));
// prints: 1199, 4037, 2065, 374, 279, 1920, 315, 45473, 264, 925, 1139, 264, 1160, 315, 11460, 13
```